ios/xcode/urlSession/src/js_Foundation/js_NSURLSessionConfiguration_P.m

```diff
@@ -1920,7 +1920,7 @@ JSValueRef backgroundSessionConfigurationForNSURLSessionConfigurationConstructor
     {
     	bool identifier$0$free = false;
     NSString * identifier$0 = HyperloopJSValueRefToNSString(ctx,arguments[0],exception,&identifier$0$free);
-    	NSURLSessionConfiguration * result$ = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier$0];
+    	NSURLSessionConfiguration * result$ = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier$0];
```

Comment:
`backgroundSessionConfigurationWithIdentifier:` was added in iOS 8. We still want this to work on iOS 7.